src/main/java/com/foodybuddy/model/Buyer.java

```diff
@@ -0,0 +1,415 @@
+package com.foodybuddy.model;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import org.hibernate.envers.Audited;
+
+
+/**
+ * The Class Buyer.
+ */
+@Entity
+@Audited
+@Table(name="Buyer")
+public class Buyer{
+	
+	/** The id. */
+	@Id
+	@Column(name="id")
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	private Integer id;
+	
+	
+	/** The apartment. */
+	@ManyToOne
+	@JoinColumn(name="apartment_id", nullable=false)
+	private Apartment apartment;
+	
+	/** The last modified by id. */
+	@Column(name="last_modified_by_id")
+	private Integer lastModifiedById;
+	
```

Comment:
Here is Sairam's comment and my reply on lastModifiedById.
https://github.com/sairamch04/FoodyBuddy/pull/44#discussion_r69387693